app/mailers/user_mailer.rb

```diff
@@ -2,6 +2,7 @@ class UserMailer < ActionMailer::Base
   add_template_helper(ApplicationHelper)
 
   def new_user(user, token)
+    return unless (AppSettings['settings.welcome_email'] == "1" || AppSettings['settings.welcome_email'] == true)
```

Comment:
Does anyone know how I might access `ApplicationController#welcome_email?`, at the moment this is not very DRY.